kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -118,7 +118,7 @@ with DAG(
                 - name: MLFLOW_RUN_ID
                   value: {% raw %}{{ task_instance.xcom_pull(key="mlflow_run_id") }}{% endraw %}
               args:
-                - 'kedro'
+                - 'kedroa'
```

Comment:
typo?

Reply:
Good catch, test leftover. Should be fine now.